.goreleaser.yml

```diff
@@ -1,11 +1,6 @@
-# This is an example .goreleaser.yml file with some sensible defaults.
-# Make sure to check the documentation at https://goreleaser.com
 before:
   hooks:
-    # You may remove this if you don't use go modules.
     - go mod tidy
-    # you may remove this if you don't need go generate
-    - go generate ./...
```

Comment:
Comments and command `go generate ./...` were part of the template but are not useful here.